src/atoms/DetailedFieldsForStudentCV/EducationInfoForStudentCV.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { AdditionalEducationTitle, EducationLanguageTitle, IStudentEducationInfoProps } from 'context';
+
+import { StyledInfoItem, StyledInfoWrapper, StyledTitle, StyledText } from './styled';
+
+interface ITitleTextFieldProps {
+	title: string,
```

Comment:
So what is the tab size? Two spaces or four? It must be the same across all project.